apps/worker/src/app/workflow/usecases/send-message/send-message-push.usecase.ts

```diff
@@ -196,6 +225,53 @@ export class SendMessagePush extends SendMessageBase {
     );
   }
 
+  private async sendPushMissingDeviceTokensError(job: JobEntity, channel: IChannelSettings): Promise<void> {
+    const raw = JSON.stringify(channel);
+    await this.createExecutionDetailsError(DetailEnum.PUSH_MISSING_DEVICE_TOKENS, job, {
+      raw,
+      providerId: channel.providerId,
+    });
+  }
+
+  private async sendNoActiveIntegrationError(job: JobEntity): Promise<void> {
+    await this.createExecutionDetailsError(DetailEnum.SUBSCRIBER_NO_ACTIVE_INTEGRATION, job);
+  }
+
+  private async sendNoActiveChannelError(job: JobEntity): Promise<void> {
+    await this.createExecutionDetailsError(DetailEnum.SUBSCRIBER_NO_ACTIVE_CHANNEL, job);
+  }
+
+  private async sendProviderError(job: JobEntity, messageId: string, raw: string): Promise<void> {
+    await this.createExecutionDetailsError(DetailEnum.PROVIDER_ERROR, job, { messageId, raw });
+  }
+
+  private async createExecutionDetailsError(
+    detail: DetailEnum,
+    job: JobEntity,
+    contextData?: {
+      messageId?: string;
+      providerId?: ProvidersIdEnum;
+      raw?: string;
+    }
+  ): Promise<void> {
+    // We avoid to throw the errors to be able to execute all actions in the loop
+    try {
+      await this.createExecutionDetails.execute(
```

Comment:
Unfortunately downstream any `ExecutionDetailsStatusEnum.FAILED` will throw an error. I find it is not a good thing but changing that would affect in all channels so it is better to handle it in a different PR.